hadoop-tools/hadoop-aws/src/test/resources/log4j.properties

```diff
@@ -75,3 +75,11 @@ log4j.logger.org.apache.hadoop.fs.s3a.s3guard.Operations=DEBUG
 
 log4j.logger.org.apache.hadoop.mapreduce.lib.output=DEBUG
 log4j.logger.org.apache.hadoop.fs.s3a.S3AStorageStatistics=INFO
+
+# Set to debug if you need to debug S3A endpoint problems.
+#log4j.logger.org.apache.hadoop.fs.s3a.DefaultS3ClientFactory=DEBUG
+
+# This causes all remote iterator stats
+# to be logged when the RemoteIterators.foreach() method is
+# invoked
+log4j.logger.org.apache.hadoop.util.functional.RemoteIterators=DEBUG
```

Comment:
maybe info level logging is better here, or this is needed to show all stats?

Reply:
( i know this is for testing )

Reply:
yeah, its too much. Sorry, must have had it set up for me and missed I was committing it